packages/electrode-archetype-react-app/arch-clap.js

```diff
@@ -142,6 +142,17 @@ function removeLogFiles() {
   } catch (e) {} // eslint-disable-line
 }
 
+function checkForCustomWebpackConfig(defaultFile) {
+  const customFilePath = Path.join(process.cwd(), "webpack.config.js");
+  return Fs.existsSync(customFilePath) ? customFilePath : webpackConfig(defaultFile);
+}
+
+function setWebpackDevProfile() {
+  const devProfileTasks = ["dev", "server-admin", "wds.dev"];
+  process.env.WEBPACK_DEV_PROFILE = devProfileTasks.includes(process.argv[2] || "") ? "true" : "false";
```

Comment:
It's not specific to dev only so naming it `WEBPACK_DEV_PROFILE` would be misleadning.  Name it `ELECTRODE_WEBPACK_PROFILE`

Reply:
modified it.